src/domain_bridge/wait_for_graph_events.hpp

```diff
@@ -150,7 +180,10 @@ class WaitForGraphEvents
   struct TopicAndCallback
   {
     std::string topic;
-    std::function<void(const QosMatchInfo)> cb;
+    std::variant<
```

Comment:
`#include <variant>`

Reply:
Done in https://github.com/ros2/domain_bridge/pull/63/commits/2eab5ce89664146c259c77e56d7a8f880dc3fc05